packages/shared/src/components/notifications/NotificationItemAttachment.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { ReactElement } from 'react';
+import { NotificationAttachment } from '../../graphql/notifications';
+
+function NotificationItemAttachment({
+  image,
+  title,
+}: NotificationAttachment): ReactElement {
+  return (
+    <div className="flex flex-row items-center p-4 mt-2 rounded-16 border border-theme-divider-tertiary">
+      <img
+        className="object-cover w-24 h-16 rounded-16"
+        src={image}
+        alt={`Preview cover of the article: ${title}`}
```

Comment:
It's not always the case. We have attachment type.
```suggestion
        alt={`Cover image of: ${title}`}
```

Reply:
I will have to replace the `image` to `preview` as the lint is not accepting it.